WordPress/src/test/java/org/wordpress/android/ui/accounts/login/LoginSiteCheckErrorViewModelTest.kt

```diff
@@ -1,58 +1,62 @@
 package org.wordpress.android.ui.accounts.login
 
-import com.nhaarman.mockitokotlin2.verify
 import kotlinx.coroutines.InternalCoroutinesApi
 import org.assertj.core.api.Assertions
 import org.junit.Before
 import org.junit.Test
-import org.mockito.Mock
 import org.wordpress.android.BaseUnitTest
 import org.wordpress.android.TEST_DISPATCHER
-import org.wordpress.android.WordPress
 import org.wordpress.android.ui.accounts.LoginNavigationEvents
+import org.wordpress.android.ui.accounts.LoginNavigationEvents.ShowInstructions
 import org.wordpress.android.ui.accounts.LoginNavigationEvents.ShowSignInForResultJetpackOnly
-import org.wordpress.android.ui.accounts.UnifiedLoginTracker
 
 @InternalCoroutinesApi
-class LoginNoSitesErrorViewModelTest : BaseUnitTest() {
-    @Mock lateinit var unifiedLoginTracker: UnifiedLoginTracker
-    @Mock lateinit var wordPress: WordPress
-    private lateinit var viewModel: LoginNoSitesErrorViewModel
+class LoginSiteCheckErrorViewModelTest : BaseUnitTest() {
```

Comment:
Suggestion (💡): I already left some testing related comment on the [accompanying PR](https://github.com/wordpress-mobile/WordPress-Android/pull/14596). Same apply to this PR as well.